include/triton/Dialect/TritonGPU/Transforms/Passes.h

```diff
@@ -13,7 +13,7 @@ std::unique_ptr<Pass> createTritonGPUCanonicalizeLoopsPass();
 
 std::unique_ptr<Pass> createTritonGPUCoalescePass();
 
-std::unique_ptr<Pass> createTritonGPUCombineOpsPass();
+std::unique_ptr<Pass> createTritonGPUCombineOpsPass(int computeCapability = 80);
```

Comment:
Do we really need a default value?

Reply:
My understanding is that setting `num_stages=2` doesn't cause anything wrong. Suppose a user wants to enable pipelining, the minimal number of stages is expected to be 2 since num_stages=1 doesn't make use of the pipeline. However, `computeCapability = 80` could result in errors if the device is sm75 or lower. Anyway, just a minor issue.

Reply:
Looks like tbl-gen requires a default constructor.